_benchmark/user-guide/running-workloads.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: default
+title: Running a workload
+nav_order: 9
+parent: User guide
+---
+
+# Running a workload
+
+Once you have a complete understanding of the various components of an OpenSearch Benchmark (OSB) [workload]({{site.url}}{{site.baseurl}}/benchmark/user-guide/understanding-workloads/anatomy-of-a-workload/), you can run your first workload. 
+
+## Step 1: Find the workload name
+
+To learn more about the standard workloads that come with OSB, use the following command:
+
+```
+opensearch-benchmark list workloads 
+```
+{% include copy.html %}
+
+A list of all workloads supported by OpenSearch Benchmark appears. Review the list and select the workload that's most similar to the use case of your cluster.
+
+## Step 2: Running the test
+
+After you've selected the workload, you can invoke the workload using the `opensearch-benchmark execute-test` command. Replace  `--target-host` with the `host:port` pairs for your cluster and `--client-options` with any authorization options required to access the cluster. The following example runs the `nyc_taxis` workload on a localhost for testing purposes. 
+
+If you want to run a test on an external cluster, see [Running the workload on your own cluster](#running-a-workload-on-an-external-cluster).
+
+```bash
+opensearch-benchmark execute-test --pipeline=benchmark-only --workload=nyc_taxis --target-host=https://localhost:9200 --client-options=basic_auth_user:admin,basic_auth_password:admin,verify_certs:false
+```
+{% include copy.html %}
+
+
+Results from the test appear in the directory set by the `--output-path` option in the `execute-test` command.
+
+### Test mode
+
+If you want to run the test in test mode to make sure your workload operates as intended, add the `--test-mode` option to the `execute-test` command. Test mode ingests only the first 1,000 documents from each index provided and runs query operations against them.
+
+## Step 3: Validate the test
+
+After an OSB test runs, take the following steps to verify that it has run properly:
+
+- Note the number of documents in the OpenSearch or OpenSearch Dashboards index that you plan to run the benchmark against.
+- In the results returned by OSB, compare the `workload.json` file for your specific workload and verify that the document count matches the number of documents. For example, based on the [nyc_taxis](https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/main/nyc_taxis/workload.json#L20) `workload.json` file, you should expect to see `165346692` documents in your cluster.
```

Comment:
Should these be numbered instead of bullets? They're referred to as "steps" above.

Reply:
Let's number them.